addons/mail/static/tests/activity/activity.test.js

```diff
@@ -1,40 +1,37 @@
-/** @odoo-module alias=@mail/../tests/activity/activity_tests default=false */
-const test = QUnit.test; // QUnit.test()
-
-import { serverState, startServer } from "@bus/../tests/helpers/mock_python_environment";
-
-import { openFormView, start } from "@mail/../tests/helpers/test_utils";
+import { describe, expect, test } from "@odoo/hoot";
 
 import { deserializeDateTime, serializeDate, today } from "@web/core/l10n/dates";
 import {
-    mockTimeout,
-    patchDate,
-    patchTimeZone,
-    patchWithCleanup,
+    assertSteps,
+    click,
+    contains,
+    createFile,
+    defineMailModels,
+    inputFiles,
+    openFormView,
+    start,
+    startServer,
+    step,
     triggerHotkey,
-} from "@web/../tests/helpers/utils";
-import { assertSteps, click, contains, createFile, inputFiles, step } from "@web/../tests/utils";
+} from "../mail_test_helpers";
+import { mockService, onRpc, patchWithCleanup } from "@web/../tests/web_test_helpers";
+import { advanceTime, mockDate } from "@odoo/hoot-mock";
 import { getOrigin } from "@web/core/utils/urls";
+import { getMockEnv } from "@web/../tests/_framework/env_test_helpers";
+import { actionService } from "@web/webclient/actions/action_service";
 
-const views = {
-    "res.fake,false,form": `
-        <form string="Fake">
-            <sheet></sheet>
-            <div class="oe_chatter">
-                <field name="message_ids"/>
-            </div>
-        </form>`,
-};
-
-QUnit.module("activity");
+describe.current.tags("desktop");
+defineMailModels();
 
 test("activity upload document is available", async () => {
     const pyEnv = await startServer();
     const partnerId = pyEnv["res.partner"].create({});
-    const [activityTypeId] = pyEnv["mail.activity.type"].search([["name", "=", "Upload Document"]]);
+    const activityType = pyEnv["mail.activity.type"]._records.find(
```

Comment:
Why the diff? pyEnv no longer works?

Reply:
Somehow `read()` and `read()` doesn't work to get the records.
I haven't checked, but could be either records are `active: false` by default or there's a bug/lack of implementation in mock server / models.